### PR TITLE
Latest R version

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
# Description
- fix version for R, to install latest version
#### Related issue:
https://github.com/actions/virtual-environments/issues/1220
## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/installers/r.sh [hunk 1]
```diff
@@ -4,7 +4,10 @@
 source $HELPER_SCRIPTS/document.sh
 
 # install R
-sudo apt-get install -y r-base
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
+sudo add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran40/'
```
Comment (miketimofeev) [inline images/linux/scripts/installers/r.sh:8]: why is it pointed to bionic only? What about ubuntu 16&20?

File: images/linux/scripts/installers/r.sh [hunk 2]
```diff
@@ -2,9 +2,22 @@
...
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
+source $HELPER_SCRIPTS/os.sh
 
 # install R
-sudo apt-get install -y r-base
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
+
+if isUbuntu16; then
+    sudo add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu xenial-cran40/'
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/r.sh:11]: `getOSVersionLabel`

Comment (dmitry-shibanov) [reply images/linux/scripts/installers/r.sh]: Done.

File: images/linux/scripts/installers/r.sh (hunk 1 above)
Comment (dmitry-shibanov) [reply images/linux/scripts/installers/r.sh]: Done.

**Pull request merged**